### ISSUE TITLE
Alt-arrows shortcuts don’t work unless address bar is focused

### ISSUE THREAD

**Issue by Trit34** [skyjake/lagrange#589]
A weird but came in 1.15.6 (and I don’t know if it is fixed in 1.15.7, not yet released at this time): Alt-arrows keyboard shortcuts (Alt-←/↑/→) don’t work to go to the previous/parent/next page, unless I move the focus on the address bar one time. Then, these shortcuts will work again as expected. Quite disturbing.

Steps to reproduce:
– Open Lagrange (I’m using the AppImage pre-built binary for Linux x86-64).
– Open any page from the bookmarks panel or stay in the current opened tab/page.
– Try to go to the parent page (Alt-↑) or the previous one in the history (Alt-←): it does nothing.
– Put the focus on the address bar (Ctrl-L).
– Try again the previous step: it works again.

Expected behavior: the shortcut should work at all times, even without select the address bar first.

Thanks for fixing it!

**Comment by skyjake** [comment]
Unfortunately, this looks like an SDL bug. The value of `SDL_IsTextInputActive()` returns 1 after launching the app even though text input has never been active. Only after activating and deactivating text input the function returns 0.

I can work around this, of course.

**Comment by sikmir** [comment]
Another problem with `SDL_IsTextInputActive`:
```sh
lagrange> /build/source/src/app.c: In function 'processEvents_App':
lagrange> /build/source/src/app.c:1916:26: error: implicit declaration of function 'SDL_IsTextInputActive'; did you mean 'SDL_IsTextInputStarted'? [8;;https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wimplicit-function-declaration-Werror=implicit-function-declaration8;;]
lagrange>  1916 |                     if (!SDL_IsTextInputActive()) {
lagrange>       |                          ^~~~~~~~~~~~~~~~~~~~~
lagrange>       |                          SDL_IsTextInputStarted
```

**Comment by skyjake** [comment]
Which version of SDL is this built against? @sikmir

**Comment by sikmir** [comment]
> Which version of SDL is this built against? @sikmir

2.26.3, that's what we have on NixOS.

**Comment by skyjake** [comment]
Does commit e8a4dad compile for you? I've removed the calls to this SDL function.

**Comment by sikmir** [comment]
> Does commit [e8a4dad](https://github.com/skyjake/lagrange/commit/e8a4dad6930d16aa0811d04d06432cd6b59b472e) compile for you? I've removed the calls to this SDL function.

Yes, that's OK now. I've added the patch https://github.com/NixOS/nixpkgs/pull/222936.

**Comment by Trit34** [comment]
Fixed in 1.15.7. Thank you!
